src/main/java/services/ReportService.java

```diff
@@ -0,0 +1,43 @@
+package services;
+
+import java.util.HashMap;
+import java.util.Map;
+import model.Position;
+import model.Storage;
+
+public class ReportService {
+    private static final String HEADER_FOR_REPORT = "PRODUCT, QUANTITY, SHELF LIFE \n";
+    private static final String FILE_PATH_TO_WRITE_RESULT = "src/test/java/results/result.txt";
```

Comment:
you can use test package for files for testing. let's use src/main/resources in this case

Reply:
done